src/Centreon/Application/Controller/AcknowledgementController.php

```diff
@@ -642,38 +666,46 @@ public function massDisacknowledgeResources(Request $request): View
         $contact = $this->getUser();
         $this->acknowledgementService->filterByContact($contact);
 
-        // Validate the content of the DELETE request against the JSON schema validator
-        $results = $this->validateAndRetrievePostData(
-            $request,
-            'Acknowledgement/DisacknowledgeResources.json'
-        );
+        $payload = (string) $request->getContent();
+        $disacknowledgementPayload = json_decode($payload, true);
```

Comment:
```suggestion
        $disacknowledgementPayload = json_decode((string) $request->getContent();, true);
```
$payload was used once